airbyte-cdk/bulk/core/load/src/testFixtures/kotlin/io/airbyte/cdk/load/test/util/destination_process/NonDockerizedDestination.kt

```diff
@@ -93,11 +91,6 @@ class NonDockerizedDestination(
     }
 }
 
-// Notably, not actually a Micronaut factory. We want to inject the actual
-// factory into our tests, not a pre-instantiated destination, because we want
-// to run multiple destination processes per test.
-@Singleton
-@Requires(notEnv = [DOCKERIZED_TEST_ENV])
```

Comment:
This is because docker-v-nondocker is now controlled by an explicit env variable check in the process factory?

Reply:
yep, exactly. I just forgot to remove these annotations in https://github.com/airbytehq/airbyte/pull/47006